auto_nag/scripts/email_nag.py

```diff
@@ -217,6 +218,11 @@ def sendMail(toaddrs, msg, username, password, dryrun=False):
 
     options, args = parser.parse_known_args()
 
+    if options.dryrun:
```

Comment:
I think you can simplify these lines with a single line : phonebook.PhonebookDirectory(options.dryrun)